subgraphs/compound-v2/src/helpers/generic.ts

```diff
@@ -0,0 +1,26 @@
+import { BigDecimal } from "@graphprotocol/graph-ts"
+
+export function exponentToBigDecimal(decimals: i32): BigDecimal {
+    let bd = BigDecimal.fromString('1')
+    for (let i = 0; i < decimals; i++) {
+      bd = bd.times(BigDecimal.fromString('10'))
+    }
+    return bd
+}
+
+export let zeroBD = BigDecimal.fromString('0')
+export let mantissaFactorBD: BigDecimal = exponentToBigDecimal(18)
+export let cTokenDecimalsBD: BigDecimal = exponentToBigDecimal(8)
+export let cTokenDecimals = 8
+
+export const unitrollerAddress = '0x3d9819210a31b4961b30ef54be2aed79b9c9cd3b'
+export const cUSDCAddress = '0x39aa39c021dfbae8fac545936693ac917d5e7563'
```

Comment:
I don't think we need cUSDC hardcoded, there are no edge cases where we need it that i can think of